lib/utils.js

```diff
@@ -1,6 +1,18 @@
 import NativePromise from 'native-promise-only'
+import pkgJson from '../package.json'
+
 const Promise = window.Promise || NativePromise
 
+export function addUserAgent (headers, appId) {
+  const originalUserAgent = headers && headers['User-Agent'] ? headers['User-Agent'] : ''
```

Comment:
nit: Maybe can simplified as `const originalUserAgent = headers?.['User-Agent'] || '';`

Reply:
Hi Sheen, thanks for the feedback.  Unfortunately this repo uses node 12, and optional chaining was only released with node 14 ([stack overflow ref](https://stackoverflow.com/a/59574160), [release notes node v14](https://github.com/nodejs/node/blob/fa8f149c0a4f1d7f9b1de64ea2a6e2d2e38143af/doc/changelogs/CHANGELOG_V14.md#update-to-v8-81)).